hal/lib/athena/PDP.cpp

```diff
@@ -17,29 +17,29 @@ static PDP* pdp[kNumPDPModules] = {nullptr};
 
 extern "C" {
 
-void HAL_InitializePDP(uint8_t module) {
+void HAL_InitializePDP(int32_t module) {
   if (!pdp[module]) {
```

Comment:
This just makes it really clear we need to do range checking.. both >= 0 and < NELEMS(pdp)